cli/tools/coverage/mod.rs

```diff
@@ -677,7 +678,8 @@ pub async fn cover_files(
       | MediaType::Mts
       | MediaType::Cts
       | MediaType::Tsx => {
-        match ps.emit_cache.get_emit_code(&file.specifier, None) {
+        let source_hash = get_source_hash(&file.source, ps.emit_options_hash);
+        match ps.emit_cache.get_emit_code(&file.specifier, Some(source_hash)) {
```

Comment:
This makes sense. While we're here, we could change this second parameter from `Option<u64>` to just `u64` to force people to provide a hash everywhere.